olad/RDMHTTPModule.h

```diff
@@ -612,6 +612,40 @@ class RDMHTTPModule {
                          unsigned int universe_id,
                          const ola::rdm::UID &uid);
 
+    std::string GetDimmerInfo(ola::http::HTTPResponse *response,
+                              unsigned int universe_id,
+                              const ola::rdm::UID &uid);
+
+    void GetDimmerInfoHandler(ola::http::HTTPResponse *response,
+                              const ola::rdm::ResponseStatus &status,
+                              const ola::rdm::DimmerInfoDescriptor &dimmer);
+
+    std::string GetDimmerMinimumLevels(ola::http::HTTPResponse *response,
+                                      unsigned int universe_id,
+                                      const ola::rdm::UID &uid);
+
+    void GetDimmerMinimumLevelsHandler(ola::http::HTTPResponse *response,
+                               const ola::rdm::ResponseStatus &status,
```

Comment:
Again

Reply:
It is not possible to both align these params and stay within an 80 character line length. Any advice on how you'd like that resolved?

Reply:
In which case drop all params to the line below (and indent by 4 off the void).